yt_dlp/options.py

```diff
@@ -1354,41 +1364,90 @@ def _dict_from_options_callback(
         '--no-split-chapters', '--no-split-tracks',
         dest='split_chapters', action='store_false',
         help='Do not split video based on chapters (default)')
+    postproc.add_option(
+        '--remove-chapters',
+        metavar='REGEX', dest='remove_chapters', action='append',
+        help='Remove chapters whose title matches the given regular expression. This option can be used multiple times')
+    postproc.add_option(
+        '--no-remove-chapters', dest='remove_chapters', action='store_const', const=None,
+        help='Do not remove any normal chapters from the file (default)')
```

Comment:
Better to remove `normal`. From the user perspective it's not clear which chapters aren't normal. This distinction is internal and stems from the fact that remove chapters and remove sponsors are handled by the same code.

Reply:
Since sponsor sections are also marked as chapter when marking, I thought this might cause confusion

Reply:
```suggestion
        help='Do not remove any chapters from the file (default)')
```